libsolidity/analysis/DocStringTagParser.cpp

```diff
@@ -169,6 +177,9 @@ void DocStringTagParser::parseDocStrings(
 
 	_annotation.docTags = DocStringParser{*_node.documentation(), m_errorReporter}.parse();
 
+	if (!m_resolverRan)
+		return;
```

Comment:
Why not assert? Is there a legitimate situation where we're fine with this analysis step being silently skipped?

Reply:
This kind of links with your other question of whether we ran the whole analysis again. This flag prevents the later parts. 
But i changed my approach now a bit so this became obsolete anyway :)